openfisca_us/variables/irs/credits/education/american_opportunity_credit.py

```diff
@@ -0,0 +1,82 @@
+from openfisca_us.model_api import *
+
+
+class american_opportunity_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "American Opportunity Credit"
+    unit = "currency-USD"
+    documentation = "Total value of the American Opportunity Credit"
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/25A#b"
+
+    def formula(tax_unit, period, parameters):
+        education = parameters(period).irs.credits.education
+        aoc = education.american_opportunity_credit
+        is_eligible = tax_unit.members(
```

Comment:
Also in Lifetime Learning Credit section since used twice
```suggestion
        person = tax_unit.members
        is_eligible = person(
```